sage-ios/SAGE/SAGE/AddSchoolDirectorTableViewController.swift

```diff
@@ -0,0 +1,132 @@
+//
+//  AddSchoolDirectorTableViewController.swift
+//  SAGE
+//
+//  Created by Sameera Vemulapalli on 11/21/15.
+//  Copyright © 2015 Cal Blueprint. All rights reserved.
+//
+
+import UIKit
+
+class AddSchoolDirectorTableViewController: UITableViewController {
+    
+    var directors: [[User]]?
+    var currentErrorMessage: ErrorView?
+    var activityIndicator: UIActivityIndicatorView = UIActivityIndicatorView(activityIndicatorStyle: .Gray)
+    
+    init() {
+        super.init(style: .Plain)
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.tableView.sectionIndexColor = UIColor.mainColor
+        self.title = "Choose Director"
+        self.tableView.tableFooterView = UIView()
+        self.tableView.sectionIndexBackgroundColor = UIColor.clearColor()
+        
+        self.view.addSubview(self.activityIndicator)
+        self.activityIndicator.centerHorizontally()
```

Comment:
you wanna put layout code in viewWillLayoutSubviews. this is because the frame isnt set for sure by the time viewDidLoad occurs